src/pages/blog/2022-05-02-idealny-stat.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Ideálny štát
+author: Dávid Jopek
+cover: /assets/government.jpeg
+tags:
+  - Esej
+---
+*Moja predstava uvažuje o podstate samotného štátu. Moja predstava uvažuje, že raz bude spoločnosť fungovať tak, že každý si bude môcť sám vybrať, čo bude so sebou a svojím majetkom robiť, niesť za tieto rozhodnutia plnú zodpovednosťa prestane byť každému nútená len tá jedna jediná možnosť.*
```

Comment:
Prosim, vrat spat tento italic text do povodneho stavu. Ak budeme chciet prvy odstavec v clankoch zvyraznovat, urobime to globalne. Dakujem :)

Reply:
Chcem na začiatok dať taký krátky výstižný popis, ktorý by sa zobrazil aj pri odkliku na článok. Je to časť z textu. Dať to aspoň do úvodzoviek alebo to šupnuť celé preč?

Reply:
Text sa bude stale zobrazovat aj po odkliku. Ale asi nerozumiem, co si tym myslel.

Prosim, bez uvodzoviek a bez italic. Nech sa zachova jednotny design, ako maju ostatne clanky.

Reply:
@DJopek tym myslel specificky toto - excerpt textu v grid view:

<img width="480" alt="Screenshot 2022-05-03 at 10 00 14" src="https://user-images.githubusercontent.com/6513576/166421162-b035c24a-1e31-45d9-bce1-6f4c09c65245.png">

Nevieme aktualne specifikovat custom excerpt, je to automaticky tahane z prveho textu z clanku.